src/views/error-view-switcher/index.tsx

```diff
@@ -54,7 +54,8 @@ function ErrorView({ statusCode }: ErrorPageProps): React.ReactElement {
 	 * Determine if this is a 404 or versioned 404 view.
 	 * For versioned views, determine the  path without version (aka latest)
 	 */
-	const versionMatches = VERSION_PATTERN.exec(asPath)
+	const versionMatches =
+		VERSION_PATTERN.exec(asPath) || TFE_VERSION_PATTERN.exec(asPath)
```

Comment:
Expanding this version matching to match URL versions for TFE (`v202401-1`) or any other docs page (e.g. `v1.0.0` or `v1.5.x`)